core-blocks/latest-comments/style.scss

```diff
@@ -0,0 +1,59 @@
+.wp-block-latest-comments {
+	padding-left: 2.5em;
+
+	&__comment-timestamp {
+		color: $dark-gray-100;
+		font-size: 12px;
+		padding: 0 2px;
+	}
+
+	li {
+		line-height: 1.1;
+	}
+
+	&.has-avatars > li {
+		min-height: 36px;
+		list-style: none;
+
+		.comment-data {
+			margin-left: 52px;
+		}
+	}
+
+	&__comment-timestamp {
+		display: block;
+		padding-top: 8px;
+	}
+
+	.wp-block-latest-comments__comment-excerpt p {
```

Comment:
Those styles should be also moved to top level. In general, there is no need to nest classes which use the same pattern.